File: contactswiththread/src/main/java/com/example/contactswiththread/Data/ContactAppDatabase.java
package com.example.contactswiththread.Data;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.example.contactswiththread.Contact;
import com.example.contactswiththread.Utils.Util;


@Database(entities = {Contact.class}, version = Util.DATABASE_VERSION)
public abstract class ContactAppDatabase extends RoomDatabase {

    public abstract ContactDAO getContactDAO();
}

File: contactsinroomlibrary/src/main/java/com/example/contactsinroomlibrary/data/ContactAppDatabase.kt
package com.example.contactsinroomlibrary.data

import androidx.room.Database
import androidx.room.RoomDatabase
import com.example.contactsinroomlibrary.Contact
import com.example.contactsinroomlibrary.utils.Util

@Database(entities = [Contact::class], version = Util.DATABASE_VERSION, exportSchema = false)
abstract class ContactAppDatabase : RoomDatabase() {
    abstract fun getContactDAO(): ContactDAO
}

